storage/key_index.go

```diff
@@ -215,8 +220,9 @@ func (ki *keyIndex) String() string {
 }
 
 type generation struct {
-	ver  int64
-	revs []reversion
+	ver     int64
+	created reversion // when the generation is created (put in first reversion).
```

Comment:
i do not understand. the version of first one in the generation is 1. the second one is 2... when you compact it, you need to have a variable to remember the version.